packages/framework/ai-collab/src/test/explicit-strategy/agentEditing.spec.ts

```diff
@@ -0,0 +1,271 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "node:assert";
+
+// eslint-disable-next-line import/no-internal-modules
```

Comment:
I'd say this rule is worth completely disabling in `.eslintrc.cjs` at this point (its current entry, which only applies to test files). I was confused because our base rule allows importing from /internal for all our packages, but I see that we're currently overriding the rule to allow `*/index.js` imports, and that seems to not merge with the base rule, but override it.